cores/esp8266/core_esp8266_i2s.c

```diff
@@ -50,10 +53,10 @@ struct slc_queue_item {
   uint32  next_link_ptr;
 };
 
-static uint32_t i2s_slc_queue[SLC_BUF_CNT-1];
+static uint32_t *i2s_slc_queue;
```

Comment:
These ptrs should be inited to null.

Reply:
They are, actually.  In ANSI-C, static variables are set to all 0 and live in BSS, even if you don't explicitly list it.  I'm happy to add in the assignment, of course, to make it absolutely clear, of course.